Platform/ArrowlakeBoardPkg/Library/Stage2BoardInitLib/SmbiosInit.c

```diff
@@ -67,6 +80,55 @@ InitializeSmbiosInfo (
     VerInfoTbl  = GetVerInfoPtr ();
     TempName[8] = 0;
 
+    // Set SMBIOS default values
+    SystemManufacturer = "Intel Corporation";
+    SystemVersion = "0.0";
+    SystemSerialNumber = "System Serial Number";
+    SystemSku = "System SKU Number";
+    SystemFamily = GetCpuName();
+
+    BaseboardManufacturer = "Intel Corporation";
+    BaseboardVersion = "1";
+    BaseboardSerialNumber = "No Board Serial Number";
+
+    CHAR8 SystemProductStrBuf[SMBIOS_STRING_MAX_LENGTH];
+    CopyMem (TempName, GetPlatformName(), sizeof (UINT64));
+    AsciiSPrint (SystemProductStrBuf, sizeof (TempStrBuf), "%a (CPU:%a)", TempName, GetCpuName ());
+    SystemProductName = SystemProductStrBuf;
+
+    CHAR8 BaseboardStrBuf[SMBIOS_STRING_MAX_LENGTH];
+    AsciiSPrint (BaseboardStrBuf, sizeof (BaseboardStrBuf), "%a (ID:%02X)", TempName, GetPlatformId ());
+    BaseboardProductName = BaseboardStrBuf;
+
+    // Load Device Info binary and Override Smbios value
+    if (FeaturePcdGet (PcdSmbiosOverride)) {
+      DEVICE_INFO_DATA     *DeviceInfo = NULL;
+      UINT32               DeviceInfoSize = 0;
+      EFI_STATUS           Status;
+
+      Status = LoadComponent (SIGNATURE_32 ('D', 'E', 'V', 'I'), SIGNATURE_32 ('D', 'I', 'N', 'F'),
+                            (VOID **)&DeviceInfo, &DeviceInfoSize);
```

Comment:
It is not necessary to add SmbiosOverrideLib, it makes SBL complicated and bigger. 
The default production values could comes from PCDs as EDK2 did. and these default values will be updated by SBL container IFWI/BIOS/NRD/IPFW/SMBS if the container exists.